src/app/utils/POAPUtils.ts

```diff
@@ -199,11 +212,21 @@ const POAPUtils = {
 			throw new ValidationError('There is not enough POAP links for all the participants!');
 		}
 		
+		const results: POAPDistributionResults = {
```

Comment:
Hey @amaredeus, this object is repeated twice in this file. It’s nominal, and as such I don’t see it being a problem to leave this as is. However, is there a possibility that this may be used elsewhere? If so, it may make sense to pull it out as top level variable.